Update image tests (#421)
atbcb/ICTTestingBaseline: _baselines/ChangeLog3.md

| Multiple Baseline Tests | Test Instructions: corrected outline numbering, added missing requirement mappings |
| 3.1 Non-Interference | Identify Content: added links to Baseline Tests |
| 5. Changing Content | Significant changes to this test to cover SC 4.1.2 more accurately. Name of test changed to 5. User Controls. |
| 6. Images | Limitations, Assumptions, Exceptions: removed "Equivalent descriptions for an image within page text could allow an image to be considered decorative. However, this does not necessitate removal of any accessible text attributes from the image." |

> | 6. Images | Test Method Rationale: added "The image tests evaluate the images as they were coded to indicate whether they are meaningful or decorative, leaving that determination to the author of the content. There are certain scenarios where that determination would be incorrect, and the test will fail." |

Reviewer: Need update to line 15

Author: What kind of update?

Author: Ah, thanks. Change made.

> | 6. Images | Test Method Rationale: added "The image tests evaluate the images as they were coded to indicate whether they are meaningful or decorative, leaving that determination to the author of the content. There are certain scenarios where that determination would be incorrect, and the test will fail." |

Author: ```suggestion
| 6. Images | Test Method Rationale: added "The image tests evaluate the images as they were coded to indicate whether they are meaningful or decorative, leaving that determination to the author of the content. However, there are certain scenarios as described in the tests where the author's determination would be incorrect." |
```
